woocommerce-payments.php

```diff
@@ -160,21 +160,32 @@ function wcpay_init_subscriptions_core() {
 		$is_plugin_active = function( $plugin_name ) {
 			$plugin_slug = "$plugin_name/$plugin_name.php";
 
+			// Check if the plugin is in the process of being activated via the Admin > Plugins screen.
 			if ( isset( $_GET['action'], $_GET['plugin'] ) && 'activate' === $_GET['action'] && $plugin_slug === $_GET['plugin'] ) { //phpcs:ignore WordPress.Security.NonceVerification.Recommended
 				return true;
 			}
 
-			if ( defined( 'WP_CLI' ) && WP_CLI && isset( $GLOBALS['argv'] ) && 4 >= count( $GLOBALS['argv'] ) && 'plugin' === $GLOBALS['argv'][1] && 'activate' === $GLOBALS['argv'][2] && $plugin_name === $GLOBALS['argv'][3] ) {
-				return true;
+			// Check if the plugin is in the process of being activated via the WP CLI.
+			if ( defined( 'WP_CLI' ) && WP_CLI ) {
```

Comment:
The `isset( $GLOBALS['argv'] )` condition has been removed as part of this change. This seems pretty low risk given every use of ` $GLOBALS['argv']` I found assumes it will be present. 

I'll leave this up to you if you want to add it back in to be safe.

Reply:
Thanks for adding these comments @james-allan 🙌 . Does `the plugin` refer to WCPay here? Might be clearer to disambiguate, e.g. `the WCPay plugin`.

Reply:
Oh .. it's generic, `$plugin_name`. So maybe `Check if specified $plugin_name is in the process of…` etc is clearer.

Reply:
> The isset( $GLOBALS['argv'] ) condition has been removed as part of this change.

Good catch, @james-allan. It doesn't hurt and more safe to check if `$GLOBALS['argv']` is set, so I added that check.

I also addressed @haszari's feedback.